### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Gagan-Deogan**
Opening PR to get code review on the backend of the e-commerce. If any face any problem in understanding feel free to contact me. Thanks in Advance
**Features**:
Multiple Quizzes
Evaluate Score
Option for review mistakes
15sec Timer for Each question
skip question

### CODE REVIEW

File: src/Context/QuizContext/reducer.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      return state.attemptedQuiz
+        ? {
+            ...state,
+            totalScore: calculateScore(state.attemptedQuiz.questions),
+          }
+        : state;
+    default:
+      return state;
+  }
+};
```
Comment (vineet-singhal2312) [inline src/Context/QuizContext/reducer.ts]: You can create two separate folder, one for your types and and one for this reducer for ex. reducer.ts  this and QuizContext.type for your types.

File: src/Pages/Home/Home.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import { useQuizzez } from "../../Context/QuizziesContext";
+
+export const Home = () => {
+  const { quizzes } = useQuizzez();
+  return (
+    <>
+      <section className="lg:container mx-auto px-3">
+        <ul className="grid sm:grid-cols-2 md:grid-cols-4 gap-4 my-4">
+          {quizzes?.map((quiz) => (
+            <Card quiz={quiz} />
```
Comment (vineet-singhal2312) [inline src/Pages/Home/Home.tsx]: you can change this component name Card to QuizCard so is more readable....!!

**Review by vineet-singhal2312 — COMMENTED** [review]
Well written code.....great job!!

File: src/Pages/Home/Home.tsx (hunk 2 above)
Comment (Gagan-Deogan) [reply src/Pages/Home/Home.tsx]: Yeah i can do so , but doesn't adding any value it anyway going have quiz details only

**Review by shubambhasin — COMMENTED** [review]
Great work !! Tried very hard to find something that can be pointed out, but couldn't !!!! Keep the good work up.

**Comment by NadaaFarook** [comment]
#1

**Pull request merged**